notebooks/spark-intro.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "95ca3df2-cad2-4e16-8768-bfca026bb576",
+   "metadata": {},
+   "source": [
+    "# Spark RDDs\n",
+    "This notebook is intended to teach you the basic structure of Apache Spark, from the basic structures to some of the more advanced techniques.  We'll do our best to give you practical advice, and not get bogged down in little minutia."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e323ed5-86bc-48aa-a85d-4c53d546d1d0",
+   "metadata": {},
+   "source": [
+    "## Initializing Spark\n",
+    "We must begin by making sure that the needed JARs are present on the system, and that the interpreter knows we're loading them into the classpath."
```

Comment:
Might be sensible to provide a parenthetical about what a JAR is (literally, just a few words about how an archive format full of class definitions is what the JVM uses and spark distributes)